src/app.js

```diff
@@ -27,6 +29,56 @@ const adminBro = new AdminBro({
     rootPath: '/admin',
 })
 
+// IEX Api
+const quote = async (sym) => {
+    const quoteData = await iex.quote(sym);
+    qd = quoteData
+
+    const market = await Market.findOne({ name: qd.primaryExchange })
+
+    if (!market) {
+        await new Market({ name: qd.primaryExchange }).save()
+    }
+
+    const stock = await Stock.findOne({ shortName: qd.symbol })
+
+    if (!stock) {
+        console.log(stock)
+        await new Stock({
+            market: market.id,
+            shortName: qd.symbol,
+            name: qd.companyName,
+            pricePerUnit: qd.latestPrice,
+            initialPricePerUnit: qd.latestPrice,
+        }).save()
+    } else {
+        stock.pricePerUnit = qd.latestPrice
+    }
+
+    console.log("Working!!")
+
+}
+
+
+const fetch = async () => {
+    const stocks = await Stock.find({})
+    for (let i = 0; i < stocks.length; i++) {
+        await quote(stocks[i].shortName)
+    }
+}
+
+// This has a slightly different form setInterval function as it runs instantaneously for the first time
+// and then runs after the defined period of time
+const noDelaySetInterval = (func, interval) => {
+    func()
+    return setInterval(func, interval)
+}
+
+// Interval(in ms) in which the stock market data is syncronised with the real data.
+interval = 60 * 1000
```

Comment:
If the interval is in milliseconds, you're making an API call every minute, not every hour. Anyway, change this so that the time interval is specified using an environment variable and isn't hardcoded in the code.